server/static/css/user-dashboard.css

```diff
@@ -13,50 +6,85 @@ body{
     width: 100%;
 }
 
-/* Set the size of the map container */
 #map {
-    flex-grow: 2;
-    height: calc(100vh - 40px); /* Full height of the viewport minus margins */
-    min-width: 70%; /* 50% of the viewport width */
-    float: left; /* Float the map container to the left */
+    height: 550px;
```

Comment:
Better practice would be to use % ages or 'ems' for css values so that its dynamic and works with different display sizes